ts/features/messages/components/MessageDetail/MessageDetailsBody.tsx

```diff
@@ -0,0 +1,81 @@
+import * as E from "fp-ts/lib/Either";
+import { useMemo, useState } from "react";
+import { ScrollView } from "react-native";
+import { useLinkTo } from "@react-navigation/native";
+import Animated, { FadeInUp } from "react-native-reanimated";
+import { Alert, Body, VSpacer } from "@pagopa/io-app-design-system";
+import I18n from "../../../../i18n";
+import { useIOSelector } from "../../../../store/hooks";
+import { isIOMarkdownEnabledForMessagesAndServicesSelector } from "../../../../store/reducers/backendStatus/remoteConfig";
+import IOMarkdown from "../../../../components/IOMarkdown";
+import { removeCTAsFromMarkdown } from "../../utils/ctas";
+import { ServiceId } from "../../../../../definitions/backend/ServiceId";
+import { generateMessagesAndServicesRules } from "../../../common/components/IOMarkdown/customRules";
+import { setAccessibilityFocus } from "../../../../utils/accessibility";
+import { MessageMarkdown } from "./MessageMarkdown";
+
+export type MessageDetailsBodyProps = {
+  messageMarkdown: string;
+  serviceId: ServiceId;
+  scrollViewRef: React.RefObject<ScrollView>;
+};
+
+export const MessageDetailsBody = ({
+  messageMarkdown,
+  serviceId,
+  scrollViewRef
+}: MessageDetailsBodyProps) => {
+  const [rawContentShown, setRawContentShown] = useState<boolean>(false);
```

Comment:
```suggestion
  const [showRawContent, setShowRawContent] = useState<boolean>(false);
```
I believe this way it would be a bit easier on the eye, what do you think?

Reply:
Done in 1373487748e119cffc42b854f009c205dc57b463